lib/widgets/content.dart

```diff
@@ -722,7 +723,15 @@ class InlineContent extends StatelessWidget {
     required this.style,
     required this.nodes,
   }) {
-    assert(style.fontSize != null);
+    assert(
+      style.fontSize != null
+      && (
+        style.debugLabel!.contains('weightVariableTextStyle')
```

Comment:
nit: just use two asserts; formatting gets simpler/shorter, and the two conditions aren't closely related anyway